ci-operator/config/openshift/jobset-operator/openshift-jobset-operator-main.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
```

Comment:
I think it is better to create 4.19 and 4.20 yaml files manually;

* openshift-jobset-operator-release-4.19.yaml
* openshift-jobset-operator-release-4.20.yaml

Reply:
Probably automation will create but I'm not sure.

Reply:
I think automation should create it, but added to be sure.